lib/tests/metadata.spec.js

```diff
@@ -273,6 +273,44 @@ describe('metadata', () => {
             determineAppDataSpy.mockClear();
         });
 
+        it('should return metadata based on the browser.options.capabilities', () => {
+            global.browser = {
+                options: {
+                    requestedCapabilities: {
+                        w3cCaps:{
+
+                        }
+                    },
+                    capabilities: {
+                        browserName: 'chrome',
+                        'cjson:metadata': {},
+                    }
+                },
+            };
+
+            const determineAppDataSpy = jest.spyOn(Metadata, 'determineBrowserData').mockReturnValue(browserMockData);
+
+            expect(Metadata.determineMetadata(WDIO6_RUNNER_STATS)).toMatchSnapshot();
+            determineAppDataSpy.mockClear();
+        });
+
+        it('should return metadata based on the browser.options.capabilities', () => {
```

Comment:
Both tests have the same name but verify a different thing, can you fix that? Then it should be good to go